### PR TITLE
#287 logger throws error when closing the session

### PR DESCRIPTION
**Pull request by SponsoredByPuma**
Removed the unregister endpoint in the member vote page. Now the Leave Button uses this endpoint. Also when closing the session as an admin, the backend now removes the member before deleting the session.

### CODE REVIEW

File: frontend/src/views/MemberVotePage.vue [hunk 1]
```diff
@@ -17,69 +17,36 @@
         </b-col>
         <b-col cols="auto">
           <session-leave-button />
-          <estimate-timer
```
Comment (stritti) [inline frontend/src/views/MemberVotePage.vue:20]: why reformatting?

File: frontend/src/views/MemberVotePage.vue [hunk 2]
```diff
@@ -17,69 +17,36 @@
...
-            :start-timestamp="timerTimestamp"
-            :pause-timer="estimateFinished || pauseSession"
-            :duration="timerCountdownNumber"
-          />
+          <estimate-timer v-if="timerTimestamp" class="mt-3" :start-timestamp="timerTimestamp"
+            :pause-timer="estimateFinished || pauseSession" :duration="timerCountdownNumber" />
         </b-col>
       </b-row>
       <b-row v-if="isMobile">
-        <mobile-story-title
```
Comment (stritti) [inline frontend/src/views/MemberVotePage.vue:30]: useless reformatting

File: frontend/src/views/MemberVotePage.vue [hunk 3]
```diff
@@ -17,69 +17,36 @@
...
-            id="flicking"
-            :options="{
+          <flicking id="flicking" :options="{
               renderOnlyVisible: false,
               horizontal: true,
               align: 'center',
               bound: false,
               defaultIndex: 0,
               deceleration: 0.0005,
-            }"
```
Comment (stritti) [inline frontend/src/views/MemberVotePage.vue:49]: useless reformatting

**Review by stritti — CHANGES_REQUESTED** [review]

File: frontend/src/views/MemberVotePage.vue (hunk 1 above)
Comment (SponsoredByPuma) [reply frontend/src/views/MemberVotePage.vue]: Sorry, didnt notice. My IDE did this on saving. My bad. 😯

**Review by KzuDemEvin — APPROVED** [review]

**Review by stritti — APPROVED** [review]

**Pull request merged**